v2/pubsub-to-snowflake/src/main/java/com/mw/pipeline/transforms/StringTransform.java

```diff
@@ -0,0 +1,15 @@
+package com.mw.pipeline.transforms;
+
+import org.apache.beam.sdk.io.gcp.pubsub.PubsubMessage;
+import org.apache.beam.sdk.transforms.DoFn;
```

Comment:
This transform is not required. Instead use readStrings method of PubsubIO

Reply:
done